packages/components/src/organisms/SlideOver/SlideOver.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react'
 import type { ReactNode } from 'react'
+import React from 'react'
 
-import { Modal } from '../../'
 import type { ModalProps } from '../../'
+import { Modal } from '../../'
 
-type Direction = 'leftSide' | 'rightSide'
-type WidthSize = 'full' | 'partial'
+export type Direction = 'leftSide' | 'rightSide'
```

Comment:
Could be only "left"and "right"?

Reply:
For what I remember, that is right. But we can review it if needed.